resource/ui/ts/models/node_status.ts

```diff
@@ -141,10 +139,70 @@ module Models {
                         m("tr", [m("td", "Ranges:"), m("td", node.range_count)]),
                         m("tr", [m("td", "Leader Ranges:"), m("td", node.leader_range_count)]),
                         m("tr", [m("td", "Available Ranges:"), m("td", node.available_range_count)]),
-                        m("tr", [m("td", "Availablility:"), m("td", this._availability(nodeId))]),
+                        m("tr", [m("td", "Availablility:"), m("td", Nodes._availability(node))]),
```

Comment:
right, but in this case you're inside a static method, so `this` should be the class instance?